src/App.tsx

```diff
@@ -28,22 +26,21 @@ const queryClient = new QueryClient();
 const App = () => {
   return (
     <ThemeProvider theme={theme}>
-      <QueryClientProvider client={queryClient}>
-        <Global styles={reset} />
-        <div css={containerStyle}>
-          <AuthProvider>
-            <BrowserRouter>
-              <Routes>
-                <Route path="/" element={<Home />} />
-                <Route path="/login" element={<Login />} />
-                <Route path="*" element={<NotFoundPage />} />
-                <Route path="/my" element={<MyPage />} />
-                <Route path="/order/:id" element={<Order />} />
-              </Routes>
-            </BrowserRouter>
-          </AuthProvider>
-        </div>
-      </QueryClientProvider>
+      <Global styles={reset} />
+      <div css={containerStyle}>
+        <AuthProvider>
+          <BrowserRouter>
+            <ToastContainer position="top-right" autoClose={3000} />
```

Comment:
소소하게 큰 숫자를 표현할 때는 numeric seprator를 사용하시면 가독성을 높일 수 있을거 같아요 거기에 더해 아래처럼 표기하면 조금 더 직관적으로 몇 초인지를 파악하기 좋을거 같네요
```suggestion
            <ToastContainer position="top-right" autoClose={3 * 1_000} />
```